background/test/app.test.js

```diff
@@ -43,38 +43,48 @@ describe('behavior of cloud function', function() {
     } catch (err) {
       console.log("Wasn't able to delete Google Cloud Functions");
     }
-    try {
-      cp.execSync(
-        `firebase firestore:delete --project ${projectId} -r translations`
-      );
-    } catch (err) {
-      console.log("Wasn't able to delete firestore project");
-    }
+    console.log('before deletion');
+    const db = new Firestore({
+      project: projectId,
+    });
+    await db
+      .collection('/translations')
+      .get()
+      .then(res => {
+        res.forEach(element => {
+          element.ref.delete();
+        });
+      });
+    console.log('after deletion');
   });
 
   it('should get the correct website', async () => {
-    return await request(app)
-      .get('/')
-      .retry(5)
-      .expect(200);
+    const body = await fetch(`${app}/`);
+    const res = await body.status;
+    console.log(res);
+    expect(res).to.equal(200);
   });
 
-  it('should ask for a translation', async () => {
-    return await request(app)
-      .post('/request-translation')
-      .type('form')
-      .send({lang: 'en', v: 'como estas'})
-      .retry(5)
-      .expect(200);
+  it('should get the correct response', async () => {
+    const params = new URLSearchParams();
+    params.append('lang', 'en');
+    params.append('v', 'como estas');
+
+    const body = await fetch(`${app}/request-translation`, {
+      method: 'POST',
+      body: params,
+    });
+    console.log(await body.text());
+    const res = await body.status;
+    expect(res).to.equal(200);
   });
 
-  it("should now contain 'como estas'", async () => {
-    return await request(app)
-      .get('/')
-      .set('Accept-Encoding', 'application/json')
-      .retry(5)
-      .end((err, res) => {
-        expect(res.text.includes('como estas'));
-      });
+  it("should now contain 'how are you'", async () => {
+    await new Promise(r => setTimeout(r, 5000));
```

Comment:
If you add a sleep like this, usually you want a big comment explaining why it's needed.  Also, 5 seconds is _a lot_.